server/apps/immich/src/api-v1/album/response-dto/album.ts

```diff
@@ -0,0 +1,30 @@
+import { AlbumEntity } from '../../../../../../libs/database/src/entities/album.entity';
+import { mapUser } from '../../user/response-dto/user';
+import { mapAsset } from '../../asset/response-dto/asset';
+import { Asset } from '../../asset/response-dto/asset';
+import { User } from '../../user/response-dto/user';
+
```

Comment:
Can we changed this file name to `album-response.dto.ts` it will make it easier to distinguish when you have dozen tabs open in the IDE or fuzzy search

Reply:
done 👍